azafea/model.py

```diff
@@ -247,6 +249,37 @@ class PostgresqlConnectionError(Exception):
 
 metadata = MetaData(naming_convention=NAMING_CONVENTION)
 Base = declarative_base(cls=BaseModel, constructor=BaseModel.__init__, metadata=metadata)
+views = {}
```

Comment:
I didn't see `views` being used anywhere.

Reply:
It’s used in [`command.py`](https://github.com/endlessm/azafea/blob/e4b799c62479e39f7e0d31267d543b0dd3a987e3/azafea/cli/commands.py#L22).